routes/api/queries/dhis/Network.js

```diff
@@ -304,11 +304,15 @@ function fetchData(organizationDetails) {
     organizationId,
     rootNode,
   } = organizationDetails;
-  return makeRequest(auth, baseURL, indicatorDict)
-    .then(data => interpretStats(data, indicatorDict))
+  return makeRequest(auth || {}, baseURL, indicatorDict)
+    .catch(() => {})
```

Comment:
Why don't you make this line return an object with an empty rows array instead.
```suggestion
    .catch(() => ({ rows: []}))
```

Reply:
Done